src/index.js

```diff
@@ -1,79 +1,67 @@
 import './style/main.css';
-import moreImage from './img/more.svg';
-import checkTodo from './eventsutils.js';
+import deleteImage from './img/delete.svg';
 
-const todos = [
-  { description: 'Do workouts', isComplete: false, index: 1 },
-  { description: 'Go to shop', isComplete: false, index: 2 },
-  { description: 'Do bath', isComplete: false, index: 3 },
-  { description: 'Read book', isComplete: false, index: 3 },
-];
+import {
+  addTodo, deleteTodo, clearAllComplele, checkTodo, updateDescription, refreshPage,
+} from './utils.js';
 
 const myWrapper = document.getElementById('list-wrapper');
+const todoInput = document.getElementById('todo-input');
+
+document.getElementById('refresh').addEventListener('click', () => refreshPage());
+
+todoInput.addEventListener('keyup', (e) => {
+  if (e.key === 'Enter') {
+    addTodo(e.target.value);
+  }
+});
```

Comment:
- [OPTIONAL] For an awesome User Experience, you can also add `click` eventListener. Why? A user might not know they are expected to press ENTER for them to add a task. This button ![unexpected-behaviour](https://user-images.githubusercontent.com/45002654/134320536-8af88bf8-f147-4283-ad51-5ef6ca2dee76.PNG) in the UI rings the bell to the user that it can be clicked to add a task, it's the first thing the user notices. Consider adding a `click` event listener to it that will enable the user to add tasks.